src/chocolatey.tests.integration/infrastructure/cryptography/CryptoHashProviderSpecs.cs

```diff
@@ -63,5 +63,55 @@ public void Should_provide_the_correct_hash_based_on_a_checksum()
                 _result.Should().Be(expected);
             }
         }
+
+        public class When_HashProvider_is_given_a_large_file_to_hash : CryptoHashProviderSpecsBase

```

Comment:
To ensure the file is deleted and not left on disk after the test is done, we should explicitly remove it after running the tests.

```
public override void AfterObservations()
{
    if (File.Exists(_filePath))
    {
        File.Delete(_filePath);
    }
}
```